Makefile

```diff
@@ -35,3 +35,6 @@ $(TESTS):
 	@echo "Running tests for $@"
 	@$(LINT) $@
 	@NODE_ENV=test $(MOCHA) $(MOCHA_OPTIONS) $@
+
+dev-server:
```

Comment:
could you update the readme to use `make dev-server`
https://github.com/Clever/components/blob/master/README.md#contributing